pkgs/servers/monitoring/cockpit/default.nix

```diff
@@ -0,0 +1,168 @@
+{ lib
+, stdenv
+, fetchzip
+, fetchurl
+, fetchFromGitHub
+, autoreconfHook
+, bash
+, coreutils
+, docbook_xml_dtd_43
+, docbook_xsl
+, findutils
+, gettext
+, git
+, glib
+, glibc
+, gnused
+, gnutls
+, json-glib
+, krb5
+, libssh
+, libxcrypt
+, libxslt
+, makeWrapper
+, nodejs
+, openssl
+, pam
+, pkg-config
+, polkit
+, python3Packages
+, ripgrep
+, systemd
+, udev
+, xmlto
+}:
+
+stdenv.mkDerivation rec {
+  pname = "cockpit";
+  version = "282.1";
+
+  src = fetchFromGitHub {
+    owner = "cockpit-project";
+    repo = "cockpit";
+    rev = "83d2ca1295a47c4d2c5a2904a27f8087d2641657";
+    sha256 = "sha256-R3IMga5/j1tmUcXaZpzo3VhgPmsn03yI7aZUnxJouVU=";
+    leaveDotGit = true;
+    fetchSubmodules = true;
+    deepClone = true;
+  };
+
+  nativeBuildInputs = [
+    autoreconfHook
+    makeWrapper
+    docbook_xml_dtd_43
+    docbook_xsl
+    findutils
+    gettext
+    git
+    glib.dev
+    libxcrypt
+    libxslt.bin
+    nodejs
+    pam.out
```

Comment:
those might be, too

Reply:
xslt stuff are only used when building docs

nodejs is only used to bundle the web stuff, not required in runtime